tests/suites/test_suite_debug.function

```diff
@@ -193,3 +193,30 @@ exit:
     mbedtls_ssl_config_free( &conf );
 }
 /* END_CASE */
+
+/* BEGIN_CASE */
+void mbedtls_debug_print_mpi_sanity ( char * file,
+                                      int line, char * prefix )
+{
+    mbedtls_ssl_context ssl;
+    mbedtls_ssl_config conf;
+    struct buffer_data buffer;
+    mbedtls_mpi val;
+
+    mbedtls_ssl_init( &ssl );
+    mbedtls_ssl_config_init( &conf );
+    mbedtls_mpi_init( &val );
```

Comment:
I'm not sure if it is worth it, but the code adds two checks, and this only tests the configuration where both are 0/NULL.

Reply:
The case that `X->p` is NULL but `X->n` is non `0` is covered in [Debug print buffer #1](https://github.com/ARMmbed/mbedtls/blob/development/tests/suites/test_suite_debug.data#L25-L26)
There seems not legit to create a `X` with `X->p` is not NULL and `X->n` is `0`.

And the test is sufficient to uncover the current issues:
*  Integer underflow when we subtract 1 from 0
for the unsigned local variable [n](https://github.com/ARMmbed/mbedtls/blob/development/library/debug.c#L235).
* crash of the program.